lib/improver/spotdata/tests/test_common_functions.py

```diff
@@ -0,0 +1,605 @@
+# -*- coding: utf-8 -*-
```

Comment:
Gavin has a point, we were using the non-camel case just for test classes. I've talked to Gavin and thought about it - provided that the name of the test class is relatively clear, I'm happy for it to be in camel case.